src/sparseml/yolov8/trainers.py

```diff
@@ -279,6 +279,10 @@ def _setup_train(self, rank, world_size):
         super()._setup_train(rank, world_size)
         # NOTE: self.resume_training() was called in ^
 
+        if rank in {0, -1}:
+            self.test_loader = self.get_dataloader(self.testset, batch_size=max(1, self.train_loader.batch_size // 4), rank=-1, mode='val')
```

Comment:
Won't this lead to batch size 0 if the original batch size is between 1 and 3? Maybe make sure that it's at least 1